src/language/index.md

```diff
@@ -156,6 +530,12 @@ keep these facts and concepts in mind:
     [exception][] being raised while the code executes.
 
 
+## Additional resources
+
+More code samples are in the the
+[library tour](/guides/libraries/library-tour)
+and the [Dart API reference,]({{site.dart-api}}).
+
```

Comment:
I'm not sure this is the best place for the note. Perhaps earlier on or at the end of **Important concepts** and still in a note-alert like before?

Reply:
Sure, I like those ideas. I'm prefer further towards the end just because I don't think Effective Dart is useful for _most_ people fresh off the intro page. It's definitely more already-in-the-weeds-developing kind of guidance